src/stores/webSocket.js

```diff
@@ -52,11 +71,19 @@ export const useWebSocket = defineStore('webSocket', () => {
       return
     }
     const parsedData = camelcaseKeysDeep(JSON.parse(data))
-    await processSocketMessage(parsedData)
+    const isSpecificEntityMessage = parsedData.subscription === 'Object'
```

Comment:
Could you explain this condition? Is it correct or was it supposed to be a typeof check?

Reply:
This is correct. You can subscribe to general entity type messages (e.g. Transactions, Microblocks...) or a specific entity e.g. transaction id. This is the second case. In this scenario the message.subscription is 'Object' and the payload is specific to the entity type you subscribed.

I struggled to figure out a good name for this so originally it was without extracting to a dedicated const.